apps/web/pages/event-types/[type]/index.tsx

```diff
@@ -283,7 +283,7 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
             if (metadata?.multipleDuration.length < 1) {
               throw new Error(t("event_setup_multiple_duration_error"));
             } else {
-              if (input.length && !metadata?.multipleDuration?.includes(input.length)) {
+              if (!input.length && !metadata?.multipleDuration?.includes(input.length)) {
```

Comment:
We want to show the error if there is no event type duration and if the duration is not included as a value. This failed when duration was undefined. Fixed.